spec/fitbank_api/pix/payment_order_spec.rb

```diff
@@ -0,0 +1,94 @@
+# typed: false
+
+require 'securerandom'
+
+RSpec.describe FitBankApi::Pix::PaymentOrder do
+  describe 'payment order' do
+    let!(:sender_bank_info) { build(:bank_info) }
+
+    let!(:receiver_pix_key) { 'test_pix_key' }
+    let!(:receiver_bank_info) do
+      # This data is taken from an example FitBank sent us and it's working.
+      # Different kinds of bank_account_digit can fail (even if they are acceptable to the API),
+      # the same goes for other props.
+      build(
+        :bank_info,
+        bank_code: '450',
+        bank_agency: '0001',
+        bank_account: '182198382',
+        bank_account_digit: '5'
+      )
+    end
+
+    let!(:credentials) { build(:credentials) }
```

Comment:
Maybe some of those values can use the lazy `let` declaration instead of `let!`.